src/storyData.js

```diff
@@ -0,0 +1,14 @@
+const storyData = [
+  {
+    title: 'One Sentence Wonder',
+    inputWords: {Adjective: '', Name: '', Verb: ''},
+    text: ['This is a ', ' story called ', ' and it ', '!']
```

Comment:
This isn't particularly flexible. You can only have one of each input word type, and you're relying on key order, which shouldn't be depended on. Probably should use [templates](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals#Tagged_templates) for `text`.